_data/guides.yaml

```diff
@@ -43,6 +43,9 @@ categories:
       - title: Using Credential Stores to Replace Clear Text Passwords in WildFly
         url: /guides/security-credential-store-for-passwords
         description: How to use credential stores to specify passwords for resources.
+      - title: Using Credential Stores With Encrypted Expressions on WildFly
```

Comment:
Small nit, the title above uses "in WildFly", this one uses "on WildFly". Maybe we could use "with WildFly" or "for WildFly" instead for both?

Reply:
fixed!

Reply:
Would be good to use the same format for the guide above too (for "Using Credential Stores to Replace Clear Text Passwords in WildFly")

Reply:
Sorry, just meant that it would be good to use the same title format for both guides. So where we have "Using Credential Stores to Replace Clear Text Passwords in WildFly" we could update that to "Using Credential Stores to Replace Clear Text Passwords with WildFly"